### PR TITLE
yeet the entire flow (fixes #174)

### LINKED ISSUE #174: Remove Flow
Flow bindings are generated because projects downstream were not yet converted to TypeScript. Recently, the last project was migrated, so this generation should be removed.

### PR DESCRIPTION
**Pull request by joelmccracken**
freckle no longer uses flow

### CODE REVIEW

File: package.json
Comment (joris974) [inline package.json:54]: I think `cpy-cli` could be removed as well. This library is often a source of CVEs

Comment (joelmccracken) [reply package.json:54]: Thanks, removed it!

**Review by joris974 — APPROVED** [review]
Looks good. I am guessing you are tagging and releasing a new version.

**Comment by joelmccracken** [comment]
> Looks good. I am guessing you are tagging and releasing a new version.

Yea I'm not sure how releasing it would work; I do note that it seems like there hasn't been a github release or tag saved since version 5.2.0, and version was 6.1.0. Advise plz? I can just do the obvious thing (use github's release functionality), but IDK if there is anything else involved .

**Comment by joelmccracken** [comment]
anyway looks like it can't merge until a @freckle/frontenders approves

**Review by jason-lieb — APPROVED** [review]

**Pull request merged**
